pandas/core/tools/datetimes.py

```diff
@@ -853,6 +857,13 @@ def to_datetime(
     ...                origin=pd.Timestamp('1960-01-01'))
     DatetimeIndex(['1960-01-02', '1960-01-03', '1960-01-04'], \
 dtype='datetime64[ns]', freq=None)
+
+    In case input is list-like and the elements of input are of mixed
+    timezones, return will have object type Index if utc parameter is not
+    passed in as True.
+
+    >>> pd.to_datetime(['2018-10-26 12:00 -0530', '2018-10-26 12:00 -0500'])
```

Comment:
ok to show the same example with utc=True as well

Reply:
Added an example with `utc=True`